newIDE/app/src/Profile/ProfileDialog.js

```diff
@@ -87,6 +87,7 @@ const ProfileDialog = ({
           <Tab label={<Trans>Games Dashboard</Trans>} value="games-dashboard" />
         </Tabs>
       }
+      maxWidth={!authenticatedUser.profile ? 'xs' : undefined}
```

Comment:
Why this set size?

Reply:
I was frustrated by the text being too large when not connected, I've fixed this by reworking the design of the CreateProfile element altogether now.